windows/RNSVG/LineView.cpp

```diff
@@ -30,18 +29,34 @@ void LineView::UpdateProperties(IJSValueReader const &reader, bool forceUpdate,
 
   __super::UpdateProperties(reader, forceUpdate, invalidate);
 }
-void LineView::CreateGeometry(UI::Xaml::CanvasControl const &canvas) {
-  auto const &resourceCreator{canvas.try_as<ICanvasResourceCreator>()};
-
-  float x1{Utils::GetAbsoluteLength(m_x1, canvas.Size().Width)};
-  float y1{Utils::GetAbsoluteLength(m_y1, canvas.Size().Height)};
-  float x2{Utils::GetAbsoluteLength(m_x2, canvas.Size().Width)};
-  float y2{Utils::GetAbsoluteLength(m_y2, canvas.Size().Height)};
-
-  auto const &pathBuilder{Geometry::CanvasPathBuilder(resourceCreator)};
-  pathBuilder.BeginFigure(x1, y1);
-  pathBuilder.AddLine (x2, y2);
-  pathBuilder.EndFigure(Geometry::CanvasFigureLoop::Open);
-  Geometry(Geometry::CanvasGeometry::CreatePath(pathBuilder));
+void LineView::CreateGeometry() {
+  auto const &root{SvgRoot()};
+
+  float x1{Utils::GetAbsoluteLength(m_x1, root.ActualWidth())};
+  float y1{Utils::GetAbsoluteLength(m_y1, root.ActualHeight())};
+  float x2{Utils::GetAbsoluteLength(m_x2, root.ActualWidth())};
+  float y2{Utils::GetAbsoluteLength(m_y2, root.ActualHeight())};
+
+  com_ptr<ID2D1DeviceContext1> deviceContext;
+  copy_to_abi(root.DeviceContext(), *deviceContext.put_void());
+
+  com_ptr<ID2D1Factory> factory;
+  deviceContext->GetFactory(factory.put());
+
+  com_ptr<ID2D1PathGeometry> geometry;
+  check_hresult(factory->CreatePathGeometry(geometry.put()));
+
+  com_ptr<ID2D1GeometrySink> sink;
+  check_hresult(geometry->Open(sink.put()));
+
+  // hollow instead of filled??
```

Comment:
What's this comment about?

Reply:
It was a self-note to double check if I wanted to use D2D1_FIGURE_BEGIN_HOLLOW instead of D2D1_FIGURE_BEGIN_FILLED below. I did not :)